frontend/src/pages/ForgotPasswordPage.tsx

```diff
@@ -0,0 +1,152 @@
+import { useState, FormEvent } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import { ModalContent } from '../components/modals/modalparts/ModalContent';
+import { ModalHeader } from '../components/modals/modalparts/ModalHeader';
+import { Footer } from '../components/Footer';
+import { paths } from '../routers/paths';
+import { Input } from '../components/forms/FormField';
+import { ReactComponent as MailIcon } from '../icons/mail_icon.svg';
+import { api } from '../api/api';
+import css from './ForgotPassword.module.scss';
+
+const classes = classNames.bind(css);
+
+export const ForgotPasswordPage = () => {
+  const { t } = useTranslation();
+  const defaultDisableTime = 30; // Seconds
+
+  const [view, setView] = useState('Send link');
+  const [email, setEmail] = useState('');
+  const [resendDisabled, setResendDisabled] = useState(false);
+  const [disableTime, setDisableTime] = useState(defaultDisableTime);
+
+  const sendEmailLink = () => {
+    api.sendPasswordResetLink(email);
+  };
+
+  const disableSending = () => {
+    setResendDisabled(true);
+
+    const countdownInterval = setInterval(() => {
+      if (disableTime > 0) setDisableTime((previous) => previous - 1);
+    }, 1000);
+
+    setTimeout(() => setResendDisabled(false), defaultDisableTime * 1000);
+    setTimeout(() => {
+      clearInterval(countdownInterval);
+      setDisableTime(defaultDisableTime);
+    }, defaultDisableTime * 1000);
+  };
+
+  const handleSend = (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    e.stopPropagation();
+    sendEmailLink();
+    setView('Link sent');
+  };
+
+  const handleResend = () => {
+    sendEmailLink();
+    setDisableTime(defaultDisableTime);
+    disableSending();
+  };
+
+  const sendLinkView = () => {
+    return (
```

Comment:
```suggestion
  const sendLinkView = () => (
```

This is somewhat a personal preference, but I find the single expression form nicer as it reduces the indentation